libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -647,16 +659,23 @@ MAV_RESULT AP_Mount::handle_command_do_gimbal_manager_pitchyaw(const mavlink_com
         return MAV_RESULT_ACCEPTED;
     }
 
-    // To-Do: handle earth-frame vs body-frame angles
-    // To-Do: handle pitch and yaw rates
     // To-Do: handle gimbal device id
 
     // param1 : pitch_angle (in degrees)
     // param2 : yaw angle (in degrees)
     const float pitch_angle_deg = packet.param1;
     const float yaw_angle_deg = packet.param2;
     if (!isnan(pitch_angle_deg) && !isnan(yaw_angle_deg)) {
-        set_angle_targets(0, pitch_angle_deg, yaw_angle_deg);
+        set_angle_target(0, pitch_angle_deg, yaw_angle_deg, (uint32_t)packet.param5 & GIMBAL_MANAGER_FLAGS_YAW_LOCK);
```

Comment:
should be:
set_angle_target(0, pitch_angle_deg, yaw_angle_deg, uint32_t(packet.param5) & GIMBAL_MANAGER_FLAGS_YAW_LOCK);

Reply:
fixed by re-using the "flag" variable slightly higher up so we can skip the conversion.